doc/api/Loading_a_Content.md

```diff
@@ -706,6 +706,61 @@ Those are the possible values for that option:
   More information about the `"RELOADING"` state can be found in
   [the player states documentation](./Player_States.md).
 
+### onAudioTracksNotPlayable
+
+_type_: `string|undefined`
+
+_defaults_: `"error"`
+
+Specifies the behavior when all audio tracks are not playable - This can occur if the
+device does not support the required audio codecs, or if the content cannot be decrypted,
+for example, due to an insufficient security level.
+
+Those are the possible values for that option:
+
+- `"continue"`: The player will proceed to play the content without audio.
+
+- `"error"`: The player will throw an error to indicate that the audio tracks could not be
+  played.
+
+<div class="note">
+
+- An event [`noPlayableTrack`](../api/Loading_a_Content) will be emitted if no audio
```

Comment:
The link is not right.

Should be something like `./PlayerEvents.md#noplayabletrack` (to check)

Reply:
Still the case